### PR TITLE
Image colorization using AI and Python

### LINKED ISSUE #7065: [Machine Learning] Image Colorization using AI and Python
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Image Colorization using AI and Python

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

 DeOldify is a Deep Learning (DL) based project for colorizing and restoring old images and video. It helps us add color old black and white photos adding life into them. The DL model uses a unique NoGAN architecture to train the model. We will use this model to convert some old black and white photos of a city adding color to them.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader should be able to:
- Learn about the Deoldify model and its architecture.
- Install the necessary dependencies.
- Clone the GitHub Repo for the model.
- Download the model.
- Perform colorization on old black and white photos 

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

DeOldify is a model that hasn't been around a long time. Not many tutorials exist on the web regarding the subject. With only a few lines of code, I will show our readers how they can use the model in their projects in the simplest way possible.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

- [DeOldify: A Deep Learning based project for colorizing and restoring old images](https://github.com/jantic/DeOldify)

### PR DESCRIPTION
**Pull request by smkinyan**
This closes #7065

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

File: content/articles/image-colorization-using-ai-and-python/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+![Nairobi Railway Station](/engineering-education/image-colorization-using-ai-and-python/nairobi-railway-station.jpg)
+
+Image of Stanley Hotel:
+
+![New Stanley hotel](/engineering-education/image-colorization-using-ai-and-python/new-stanley-hotel.jpg)
+
+Image of the Norfolk Hotel:
+
+![Norfolk hotel](/engineering-education/image-colorization-using-ai-and-python/norfolk-hotel.jpg)
```
Comment (wilkistermumbi) [inline content/articles/image-colorization-using-ai-and-python/index.md:110]: Please add the image sources.

Comment (smkinyan) [reply content/articles/image-colorization-using-ai-and-python/index.md]: Done

**Comment by wilkistermumbi** [comment]
@smkinyan This tutorial is very educative. It's amazing how you can achieve such results using only a few lines of code. Looking forward to more content like this. The PR will now proceed for final review.

**Comment by LinusMuema** [comment]
Hi @smkinyan 

This is indeed a great article and the tech is pretty amazing. Awesome stuff

cc: @wilkistermumbi

**Pull request merged**